actions/WalletActions.js

```diff
@@ -0,0 +1,49 @@
+import * as Wallet from '../app/services/wallet'
+
+export const ACTIONS = {
+  LOAD_WALLET_BALANCE: 'balance/load',
+  SET_USD_RATE: 'usdRate/load',
+}
+
+/**
+ * @typedef {object} WalletBalance
+ * @prop {string} confirmedBalance
+ * @prop {string} pendingChannelBalance
+ * @prop {string} channelBalance
+ */
+
+/**
+ * Fetches the Node's info
+ * @returns {import('redux-thunk').ThunkAction<Promise<WalletBalance>, {}, {}, import('redux').AnyAction>}
+ */
+export const getWalletBalance = () => async dispatch => {
```

Comment:
should be periodically updated or obtained from socket, not asked to be fetched externally

Reply:
I'm not sure what do you mean by that